backend/src/main/kotlin/app/ehrenamtskarte/backend/common/database/Database.kt

```diff
@@ -51,14 +51,16 @@ class Database {
                 config.postgres.url,
                 driver = "org.postgresql.Driver",
                 user = config.postgres.user,
-                password = config.postgres.password
+                password = config.postgres.password,
+                databaseConfig = if (config.production) {
+                    null
+                } else {
+                    DatabaseConfig.invoke {
+                        this.sqlLogger = StdOutSqlLogger
+                    }
+                },
             )
-
             transaction {
-                if (!config.production) {
-                    addLogger(StdOutSqlLogger)
```

Comment:
As this is removed, how are queries logged?

Reply:
I removed logging for this specific transaction, but added logging for all transactions by adding to the DatabaseConfig